### DISCUSSION TITLE
Preview deployment load "Comment" plugin failed

### DISCUSSION THREAD

**Discussion by ethanyu99** [vercel/community#6050] [Help]
### Summary

I migrated a nextjs project to vercel, but the "comment" function cannot be used in the preview environment of this project.

[project url](https://novita-home-git-feat-upgrade-novita-ai.vercel.app)

Compared with other projects that can run normally, I saw the following error in the browser.
![image](https://github.com/vercel/community/assets/77231520/652ff2ee-2180-448b-b29c-37b25b04a101)

I tried several times and couldn't fix him.

Help 🙏🙏🙏

### Steps to Reproduce

OPEN this project [vercel project link](https://novita-home-git-feat-upgrade-novita-ai.vercel.app)

**Comment by AntnheD** [comment]
check failure on the website (https://novita-home-qi1-feat-upgrade-novita-t2a-ai.vercel.app/), you may need to investigate why the verification process for topics is failing. This could involve checking the functionality that handles topic attestation and ensuring that it is working correctly. Look for any errors in the code logic or network requests that could be causing the failure.
